src/Image/Image.js

```diff
@@ -4,21 +4,28 @@ import PropTypes from 'prop-types';
 // Ideally, this component will take in an image object formatted by our images API and spit out an image with a proper srcset. However, I also thought I should provide a couple of fallback options, in case you want to use an image from somewhere else entirely: fallbackSrcSet and fallbackSrc. The last one will just create a normal img tag, so I really don't recommend it.
 
 function generateSrcSet(imageProps, props) {
-  if (props.image.aspect_ratios[props.aspectRatio] === null) {
-    imageProps.srcset = '';
-    return;
-  }
-
-  props.image.aspect_ratios[props.aspectRatio].instances.forEach(
-    (image, i, dataSet) => {
-      let set = `${image.url} ${image.width}w`;
-      if (i !== dataSet.length - 1) {
-        set = set.concat(',');
-      }
+  let aspectRatio = 'uncropped';
 
-      imageProps.srcSet = imageProps.srcSet.concat(set);
+  if (props.image.aspect_ratios) {
+    if (props.aspectRatio in props.image.aspect_ratios) {
```

Comment:
For some reason I forgot that `in` was a thing and that it's been in javascript forever. I need to remember this in the future!

Reply:
Yeah! It ended up being key to me setting up all the safety railings again!